go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
 	go.uber.org/mock v0.4.0
+	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc
```

Comment:
Not sure if we want the extra dependency for a simple one-off function of getting keys from a map into a slice. If we had more use of the module, then I wouldn't think about it.